specs/latest/1.0/index.html

```diff
@@ -877,6 +878,21 @@ <h4>Context creation parameters</h4>
               set to <code>false</code>, with the knowledge that a reduced-fidelity rendering mode
               should be used to improve performance.
             </dd>
+        <dt> <span class=prop-value>lowLatency</span></dt>
+            <dd>
+              <p>
+                If the value is true, the user agent will attempt to optimize the rendering of the
+                canvas to reduce latency, including potentially rendering from the front buffer.
+                This might desynchronize the canvas's display from the rest of the page and induce
```

Comment:
What's the general proposed design here? There shouldn't be much else on the page moving, so this shouldn't be wildly different from a well-optimized same-frame compositing path.

Reply:
See the comment above for how this is currently implemented in Chromium. The use of the scanout buffer bypasses the browser's usual compositing pipeline and can break some guarantees about synchronization of content on the page, but does significantly reduce latency.